python/ray/serve/api.py

```diff
@@ -235,11 +238,11 @@ def get_backend_config(backend_tag):
 
 
 @_ensure_connected
-def create_backend(backend_tag,
-                   func_or_class,
+def create_backend(backend_tag: str,
+                   func_or_class: Callable,
```

Comment:
`Union[Callable, Type[Callable]]`. `Callable` for functions and `Type[Callable]` for classes that implement `__call__`